index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE HMTL>
+<html>
+
+<head>
+<link rel=stylesheet type="text/css" href="style.css">
+  <title>Tic Tac Toe</title>
+
+  <meta charset="utf8">
+</head>
+<body background = "image.jpeg">
+
+  <header>
+    <body>
+  <h1>The Ultimate Tic Tac Toe Challenger</h1>
+
+  </header>
+
+<main>
+<form>
+
+  <input type = "hidden" name = "mover" value = "x">
+
+    <input id= "A1" input type="button" class="square" onclick="playerMove">
+    <input id= "B1" input type="button" class="square" onclick="playerMove">
+    <input id= "C1" input type="button" class="square" onclick="playerMove"><br/>
+
+    <input id= "A2" input type="button" class="square" onclick="playerMove">
+    <input id= "B2" input type="button" class="square" onclick="playerMove">
+    <input id= "C2" input type="button" class="square" onclick="playerMove"><br/>
+
+    <input id= "A3" input type="button" class="square" onclick"playerMove">
+    <input id= "B3" input type="button" class="square" onclick="playerMove">
+    <input id= "C3" input type="button" class="square" onclick="playerMove"><br/>
+
+
+    <div id="reset">
+            <button id="resetScore" onClick="clearBoard()">Reset Game</button>
+
+    </div>
+
+
+</form>
+
+</main>
+
+  <footer>
+    <p></p>
+    <p></p>
+    <p></p>
+    <p></p>
+    <p></p>
+    <p></p>
+    <p></p>
+    <p></p>
+    <p></p>
+    <p></p>
+    <p></p>
+    <p></p>
+    <p><font color="red">Lisa's Homework &copy; 2015</font> </p>
```

Comment:
No `font` tag. You're mixing stylistic tags with semantic tags, which is bad. Refer to the tags/attributes you're not allowed to use:

https://github.com/ga-dc/html_resume